data/data.json

```diff
@@ -0,0 +1,98 @@
+  },
+  {
+  "id": 6,
+  "title": "Pumpkin",
+  "category": "safe",
+  "details": "Pumpkins is similar to marrows. It contains a lot of vitamins and carotine. Should be cut onto pieces before feeding.",
+  },
+  {
+  "id": 7,
+  "title": "Marrow",
+  "category": "with_care",
+  "details": "Marrow is a nutritious vegetable. It contains a lot of water, which is good for the gastro-intestinal tract. Note, however, that because of this the amount of plain water fed to the animal should be decreased. Marrow is usually fed during summer and fall.",
+  },
+  {
+  "id": 8,
+  "title": "Cabbage",
+  "category": "with_care",
+  "details": "Cabbage is eaten by rabbits with much pleasure, but it should be fed in limited amounts. This is because it causes swelling. The animal should be accustomed to this food gradually. Cabbage contains a lot of vitamins, water, fibers and microelements needed for active growth and reproduction.",
+  },
+  {
+  "id": 9,
+  "title": "Beet",
+  "category": "safe",
+  "details": "Only sugare of forage beet is edible. It improves the structure of blood. It contains such useful substances as sugar, proteins, fats and microelements. 50 grams on one adult rabbit per day is enough.",
+  },
+  {
+  "id": 10,
+  "title": "Strawberry",
+  "category": "safe",
+  "details": "Strawberry contains a lot of vitamins and is extraordinary luscious. Can be fed as an enitre shrub.",
+  },
+  {
+  "id": 11,
+  "title": "Respberry",
+  "category": "safe",
+  "details": "Raspberry is low on calories and is rich with vitamins. Decreases temperature, restores appetite. Can be fed as an entire shrub.",
+  },
+  {
+  "id": 12,
+  "title": "Cranberry",
+  "category": "safe",
+  "details": "Cranberry boosts the immunity and cures bladder inflammation.",
+  },
+  {
+  "id": 13,
+  "title": "Blackberry",
+  "category": "safe",
+  "details": "Blackberry cures inflammation of respiratory ways. Contains tannins and vitamins A, C.",
+  },
+  {
+  "id": 14,
+  "title": "Blueberry",
+  "category": "safe",
+  "details": "Blueberry is low on calories, rich of minerals. Causes diarrhea if fed too much. Can be fed as an entire shrub.",
+  },
+  {
+  "id": 15,
+  "title": "Grape",
+  "category": "safe",
+  "details": "Grapes contain lots and lots of sugers. Berries are fed without seeds.",
+  },
+  {
+  "id": 16,
+  "title": "Melon",
+  "category": "safe",
+  "details": "Melon possesses diuretic effect, removes fever. Berries are fed without seeds.",
+  },
+  {
+  "id": 17,
+  "title": "Banana",
+  "category": "with_care",
+  "details": "Bananas contains a lot of fibers, potassium and magenesium, vitamins B needed for healthy and beautiful fur, good appetite, quick growth and mental stability. The banana peels are preffered to be fed, since they contain more fibers. Too much of this fruit intails some risks: digestion disorder, overweigth, and risk of poisonning via chemicals used during banana's processing.",
+  },
```

Comment:
Please, no trailing commas. Not sure, If I can parse this with some ready-to go parser

Reply:
Fixed.